textdb/textdb-sandbox/src/main/java/edu/uci/ics/textdb/sandbox/AlchemyAPIexample/api_key.txt

```diff
@@ -0,0 +1 @@
+c5aa33f495d42e55a13f07511e0b22825a727bf3
```

Comment:
@fukoyui Are we allowed to put this key here (in public)?  If not, we remove it, and we will put the key in our internal files (on google drive)